cylc/flow/scripts/clean.py

```diff
@@ -192,7 +192,10 @@ async def run(*ids: str, opts: 'Values') -> None:
         prompt(workflows)  # prompt for approval or exit
 
     for workflow in sorted(workflows):
-        init_clean(workflow, opts)
+        try:
+            init_clean(workflow, opts)
+        except Exception as exc:
```

Comment:
(Probably one of the few situations where it's fine to catch all exceptions IMO - but can try to be specific if needed.)

Reply:
I think this is fine here, sometimes you need to catch anything.